springboard/views/base.py

```diff
@@ -0,0 +1,50 @@
+import os
+
+from elasticgit.search import SM
+
+from springboard.utils import parse_repo_name
+
+from unicore.content.models import Category, Page
+
+from slugify import slugify
+
+
+class SpringboardViews(object):
+
+    def __init__(self, request):
+        self.request = request
+        self.language = request.locale_name
+        self.settings = request.registry.settings
+        es_host = self.settings.get('es.host', 'http://localhost:9200')
+        self.es_settings = {
+            'urls': [es_host]
+        }
+
+        repo_dir = self.settings.get('unicore.repos_dir', 'repos')
+        repo_names = map(
+            lambda repo_url: parse_repo_name(repo_url),
+            self.settings['unicore.content_repo_urls'].strip().split('\n'))
```

Comment:
We grew a `config_list()` function for this in [`springboard.utils`](https://github.com/universalcore/springboard/blob/develop/springboard/utils.py#L44).